docs/streams/stream-tasks/manage-readers.md

```diff
@@ -0,0 +1,171 @@
+---
+sidebar_position: 110
+title: Manage Readers
+---
+
+Global Data Network (GDN) stream readers are similar to stream consumers, but there are two crucial differences between them:
+
+- Readers allow you to specify the starting point for processing messages within a stream. In contrast, consumers always begin with the latest available unacknowledged message.
```

Comment:
Same point which Stoyan mentioned above. (earliest and latest unacknowledged message applies here \ too I guess)